hack-this-site/edit-a-page.md

```diff
@@ -1,14 +1,19 @@
 ---
 title: Edit a post or page
-description: All you need is a GitHub account to write a blog post or suggest edits to any page.
+description: >-
+  All you need is a GitHub account to write a blog post or suggest edits to any
+  page.
 weight: 1
+published: true
```

Comment:
this gets added automatically by prose :/

There's a ticket open for it, but is it acceptable for now?

Reply:
I _think_ so, `published` is not a known post-level variable in Hugo so it would just get treated (and ignored) as custom metadata. 

FWIW Hugo uses `draft: [false|true]` instead of `published: [true|false]`